src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Errors/UserStringBuilder.cs

```diff
@@ -397,10 +404,12 @@ protected void ErrAppendProperty(PropertySymbol prop, SubstContext pctx)
                 ErrAppendName(prop.name);
             }
         }
-        protected void ErrAppendEvent(EventSymbol @event, SubstContext pctx)
+
+        private void ErrAppendEvent(EventSymbol @event, SubstContext pctx)
         {
         }
```

Comment:
There are a few nop methods like this. Considering that part of the cause of #15464 was a check for an error condition that wasn't implemented, each of these could be the source of a bug, and need investigating.